packages/dev/core/src/PostProcesses/postProcess.ts

```diff
@@ -672,6 +676,7 @@ export class PostProcess {
             textureFormat = options.textureFormat ?? Constants.TEXTUREFORMAT_RGBA;
             shaderLanguage = options.shaderLanguage ?? ShaderLanguage.GLSL;
             uniformBuffers = options.uniformBuffers ?? null;
+            extraInitializations = options.extraInitializations ?? undefined;

```

Comment:
Nullish coalescing operator doesn't seem necessary here, since options.extraInitializations is already undefined or a function.